src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -129,4 +129,49 @@ tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
 template <size_t SpatialDim, typename Frame, typename DataType>
 tnsr::ijj<DataType, SpatialDim, Frame> deriv_spatial_metric(
     const tnsr::iaa<DataType, SpatialDim, Frame>& phi) noexcept;
+
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes time derivative of the spatial metric.
+ *
+ * \details Let the generalized harmonic conjugate momentum and spatial
+ * derivative variables be \f$\pi_{ab} = -n^c \partial_c \psi_{ab} \f$ and
+ * \f$\phi_{iab} = \partial_i \psi_{ab} \f$. As \f$ n_i \equiv 0 \f$, the time
+ * derivative of spatial metric is given by the spatial sector of the time
+ * derivative of the spacetime metric, i.e.
+ * \f$ \partial_0 g_{ij} = \partial_0 \psi_{ij} \f$.
+ *
+ * To compute the latter, we use the evolution equation for \f$ \psi_{i,j} \f$,
+ * c.f. eq.(35) of "A New Generalized Harmonic Evolution System" by
+ * Lindblom et. al, <https://arxiv.org/abs/gr-qc/0512093>:
+ * \f{align*}
+ * \partial_0 \psi_{ab} =& (1+\gamma_1) N^k \phi_{kab} - \gamma_1 N^k \phi_{kab}
```

Comment:
the first `\phi_{k a b}` on the right-hand side of the equation should be a `\partial_k \psi_{a b}` from the paper cited.